core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -1417,29 +1416,45 @@ short ExExeUtilHBaseBulkLoadTcb::work()
               masterGlob->getStatement()->getContext()->setSqlParserFlags(0x20000);
             }
           }
+        ComDiagsArea *diagsArea = getDiagsArea();
 
         cliRC = cliInterface()->executeImmediate(loadQuery,
             NULL,
             NULL,
             TRUE,
-            &rowsAffected_);
-          if (parserFlagSet)
-            masterGlob->getStatement()->getContext()->resetSqlParserFlags(0x20000);
+            &rowsAffected_,
+            FALSE,
+            diagsArea);
 
+        if (parserFlagSet)
+            masterGlob->getStatement()->getContext()->resetSqlParserFlags(0x20000);
         if (cliRC < 0)
         {
           rowsAffected_ = 0;
           cliInterface()->retrieveSQLDiagnostics(getDiagsArea());
           step_ = LOAD_END_ERROR_;
           break;
         }
-        
-        step_ = COMPLETE_BULK_LOAD_;
+        else {
+           step_ = COMPLETE_BULK_LOAD_;
+           ComCondition *cond;
+           Lng32 entryNumber;
+           while ((cond = diagsArea->findCondition(EXE_ERROR_ROWS_FOUND, &entryNumber)) != NULL) {
+              errorRowCount = cond->getOptionalInteger(0);
+              diagsArea->deleteWarning(entryNumber);
+           }
+           // Need to clear the diags Area to get the correct rowsAffected for the LOAD COMPLETE command
+           // Hence, we might lose any other warnings created at the time of loading like error during
+           // logging error rows
+           diagsArea->clear();
```

Comment:
Any warning diags could be lost.  The warning were lost before this change too. In fact, the warnings never reach the caller of the executeDirect of any SQL command called with the executor utility commands.  This PR has the fix for this. However, when I attempted to pass this warning to the executor utility command,  I was getting wrong rows loaded  for the load completion command for reasons that needed to debugged further.  Hence, as a interim solution I have cleared the diagnostics after getting the error row count and it behaves the same way as before.

When I have a solution for this anomaly, I will create a new PR